client/lib/plans/features-list.tsx

```diff
@@ -547,8 +548,16 @@ export const FEATURES_LIST: FeatureList = {
 
 	[ WPCOM_FEATURES_PREMIUM_THEMES_UNLIMITED ]: {
 		getSlug: () => WPCOM_FEATURES_PREMIUM_THEMES_UNLIMITED,
-		getTitle: () => i18n.translate( 'Unlimited premium themes' ),
-		getDescription: () => {
+		getTitle: ( { planSlug = undefined } = { planSlug: undefined } ) => {
+			if ( planSlug && isWooExpressPlan( planSlug ) ) {
```

Comment:
I was a little confused about this comment because I was certain I tested on a trial site 😆
The planSlug param is the plan is is being displayed, not the plan the site is currently on. Hence the table shows correctly since we are upgrading to a isWooExpressPlan

Reply:
Ah, I got myself confused!

Thanks, your code looks good!